sdks/python/apache_beam/runners/worker/worker_id_interceptor.py

```diff
@@ -48,17 +49,23 @@ def __init__(self, worker_id=None):
     if worker_id:
       self._worker_id = worker_id
 
+  def intercept_unary_unary(self, continuation, client_call_details, request):
+    return self._intercept(continuation, client_call_details, request)
```

Comment:
Hmm, each RPC will need to pass the worker_id which is a minor argument for bidi stream but not worth the complexity of it compared to the unary RPC. We also don't expect this to happen a lot.

Reply:
Removing the worker id means that process bundle descriptors need to be unique per control server instance.